src/SDKs/Azure.ApplicationModel.Configuration/data-plane/README.md

```diff
@@ -1,25 +1,62 @@
-# Overview 
+# Azure App Configuration client library for .NET
+Azure App Configuration is a managed service that helps developers centralize their application configurations simply and securely.
 
-Azure.ApplicationModel.Configuration is a component of the .NET Azure SDK. 
-It provides APIs for Microsoft [Azure's Application Configuration Service](https://docs.microsoft.com/en-us/azure/azure-app-configuration/) which is a service that allows to easily store and manage all application settings in one central place that is separated from code.
+Modern programs, especially programs running in a cloud, generally have many components that are distributed in nature. Spreading configuration settings across these components can lead to hard-to-troubleshoot errors during an application deployment. Use App Configuration to store all the settings for your application and secure their accesses in one place.
 
-Developers can use this SDK to interact with the [Configuration Store](https://docs.microsoft.com/en-us/azure/azure-app-configuration/quickstart-dotnet-core-app#create-an-app-configuration-store) where the configuration settings are stored.
+Use the client library for App Configuration to:
+- Create a Configuration Setting
+- Retrieve a Configuration Setting
+- Update an existing Configuration Setting
+- Delete a Configuration Setting
```

Comment:
I'm not sure it's worth splitting out CRUD operations on Configuration Settings across different lines. Could consider folding it all into one sentence: "Use the client library for App Configuration to create and manage application configuration settings".